build-libs/__tests__/build-beta-opt-in-redirect.test.ts

```diff
@@ -2,25 +2,121 @@ import buildBetaProductOptInRedirect from '../build-beta-opt-in-redirect'
 
 describe('buildBetaOptInRedirect', () => {
 	test('builds a redirect definition for beta opt-in', () => {
-		expect(
-			buildBetaProductOptInRedirect('waypoint', ['docs', 'commands', 'plugins'])
-		).toMatchInlineSnapshot(`
-      Object {
-        "destination": "https://developer.hashicorp.com/waypoint/:base/:path*",
-        "has": Array [
-          Object {
-            "key": "waypoint-io-beta-opt-in",
-            "type": "cookie",
-            "value": "true",
-          },
-          Object {
-            "type": "host",
-            "value": "(www\\\\.waypointproject\\\\.io|test-wp\\\\.hashi-mktg\\\\.com)",
-          },
-        ],
-        "permanent": false,
-        "source": "/:base(docs|commands|plugins)/:path*",
-      }
-    `)
+		expect(buildBetaProductOptInRedirect()).toMatchInlineSnapshot(`
```

Comment:
This snapshot contains the redirect definitions for all products, as all the products are present in the development config.

There is no risk to shipping these redirect rules sooner, as it would require someone to manually set the cookie if we don't have the banner showing yet.